learn/advanced/known_limitations.md

```diff
@@ -112,3 +112,15 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 **Limitation:** Meilisearch might throw an internal error when indexing large batches of documents.
 
 **Explanation:** Indexing a large batch of documents, such as a JSON file over 3.5GB in size, can result in Meilisearch opening too many file descriptors. Depending on your machine, this might reach your system's default resource usage limits and trigger an internal error. Use [`ulimit`](https://www.ibm.com/docs/en/aix/7.1?topic=u-ulimit-command) or a similar tool to increase resource consumption limits before running Meilisearch. For example, call `ulimit -Sn 3000` in a UNIX environment to raise the number of allowed open file descriptors to 3000.
+
+## Maximum database size
+
+**Limitation:** The maximum size of an index is 500GiB, and the maximum size of the task database is 10GiB.
+
+**Explanation:** Meilisearch allocates all the virtual memory it requires upfront. The database maximum size ensures instances can contain several large indexes without reaching OS limits on the amount of virtual memory available to a single process.
```

Comment:
```suggestion
**Explanation:** Meilisearch allocates all the virtual memory it requires upfront. The maximum database size ensures instances can contain several large indexes without reaching operating system limits on the amount of virtual memory available to a single process.
```
We use "operating system" instead of OS everywhere else